### DISCUSSION TITLE
GPX Track recording also if now navigation is running

### DISCUSSION THREAD

**Discussion by mkrezdo** [devemux86/cruiser#639] [Q&A] [answered]
Hi,

I was on the road last week. I drove arround 2000 km with Cruiser over 6-8 days. I made one long gpx recording and startet a new track section each day. Now I loaded the track the first time on the pc and I've seen that there are gaps if I drove some distances without running navigation (perhaps because the route ended but I decided to drive 50 km more to the next camping or because of a bigger route change because of blocked roads and I did stop navigation instead of recalculating).

I've seen already during riding that both - gpx track and day track do stop counting the km if I stop the navigation.
I think the counting schould go on during I'm still driving. Because now I've two problems. The track shows not the full track I was driving and the day/complete km are wrong.

Is it like that or do I use it not correct?

Marco

**Comment by devemux86** [comment, marked as answer]
Hi @mkrezdo,

GPS recording has in "Settings | GPS recording | Automatic recording of track" the options:
- [ ] Navigation
- [ ] Follow location

If you enable any of them, then the GPS recording starts and stops automatically when navigation / follow starts and stops.

> if I drove some distances without running navigation

If you want the GPS recording to start / stop when you want (regardless of the navigation / follow),
then you can disable these options and start / stop the GPS recording manually via its main menu.

**Reply by mkrezdo** [reply]
> GPS recording has in "Settings | GPS recording | Automatic recording of track" the options:
> 
> * [ ]  Navigation
> * [ ]  Follow location
> 
> If you enable any of them, then the GPS recording starts and stops automatically when navigation / follow starts and stops.

OK then I missunderstood this. I thought if BOTH are active, that it'll record then during "Navigation" and during "follow loction = no navigation running"

But what is then, if I activate "Follow location". It does not track my during it follows me :-) What is it doin then?



> If you want the GPS recording to start / stop when you want (regardless of the navigation / follow),
> then you can disable these options and start / stop the GPS recording manually via its main menu.

I've to try this. I think this manual mode will be the better one for me. My target is to really track my complete journey. I think I've to stop it then, if I stay somewhere over night

**Reply by devemux86** [reply]
@mkrezdo 

Follow location is activated manually with long press on the location button at bottom right.
Then if the option is enabled in GPS recording settings, it will also record the track.

If you want the follow location to activate automatically when navigation ends at destination (not manually stopped),
you can enable the option in "Settings | Navigation | Advanced | Follow after arrival".
Then if the option is enabled in GPS recording settings, it will also record the track.

**Reply by devemux86** [reply]
@mkrezdo 

Or, as you wrote, turn on / off GPS recording manually, as then you have full control.

**Reply by devemux86** [reply]
@mkrezdo 

Remember you can save your recorded track in "My collection" via the GPS recording menu,
clear the GPS recording and start a new recording.

**Reply by mkrezdo** [reply]
> @mkrezdo
> 
> Or, as you wrote, turn on / off GPS recording manually, as then you have full control.

Yes - I think that'll be the solution for me. I want to save the full track - does not matter if navigation is running or not. I use Barbuttons so I often stop navigation for some time to check sth in the map or drive a few km in a diffrent direction to a fuel stop etc. So I really need this "all the time recording". I missunderstood the follow me function ....


But I can say after this longer trip that the Cruiser app is really cool now - it provides so much good features. I really love it to navigate my tracks I've planned in kurviger. I did not miss any feature I guess

Only thing that is missing is traffic and a rain radar. But I know that this is not really possible :-)

**Comment by bios4** [comment]
> I made one long gpx recording and **startet a new track section each day**.

Might be a silly question, but how is this ("start a new track section") done in Cruiser?

**Reply by devemux86** [reply]
@bios4 

It is a new track segment `trkseg` in the recorded track `trk`.

**Reply by bios4** [reply]
Yes, but what do I need to do in Cruiser to add this to the recorded track? 😉
Click the "Pause" panel, twice?

**Reply by devemux86** [reply]
@bios4 

- You can stop and restart the GPS recording via its panel or menu.
- Or it is done automatically when navigation / follow stops and restarts.
(if the option in GPS recording settings is enabled)
- There is also the "Start new section" in the GPS recording menu.

**Reply by bios4** [reply]
OK I see, will give this a try during my next ride...

> Or it is done automatically when navigation / follow stops and restarts.
(if the setting is enabled)

Which setting are you referring to exactly?
I can't find anything like that in Settings > GPS recording.

> There is also the "Start new section" in the GPS recording menu.

With an active navigation I only get the "Stop GPS recording" button (which - in reality - is a "pause" recording, correct?) in the menu.
Nothing more...

<img src="https://github.com/user-attachments/assets/b53de4ca-cd5f-4c18-919b-b3c9acd775ab" width="33%">

**Reply by devemux86** [reply]
@bios4 

> Which setting are you referring to exactly?

"Settings | GPS recording | Automatic recording of track"

**Reply by bios4** [reply]
Ah OK that's what you meant... 👍

I thought that there was an explicit "automatically split recording in sections" setting somewhere that I didn't see so far...
